src/pages/projects/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import Head from 'next/head';
 import { client } from 'client';
-import appConfig from '../app.config';
+import appConfig from '../../app.config';
```

Comment:
You could import `appConfig` from the `baseUrl`:

```suggestion
import appConfig from 'app.config';
```